pkg/fortify/fortify.go

```diff
@@ -90,6 +91,8 @@ func NewSystemInstance(serverURL, apiEndpoint, authToken string, timeout time.Du
 func createTransportConfig(serverURL, apiEndpoint string) *ff.TransportConfig {
 	scheme, host := splitSchemeAndHost(serverURL)
 	host, hostEndpoint := splitHostAndEndpoint(host)
+	hostEndpoint = strings.TrimRight(hostEndpoint, "/")
+	apiEndpoint = strings.Trim(apiEndpoint, "/")
```

Comment:
this is not self-explanatory, imo. 

`strings.Trim("/a/b/c/", "/")` produces `a/b/c`

What is the type of input we expect here?

Reply:
I don't understand the concern. What you expect is what was meant. The `apiEndpoint` is a user configurable parameter. It is used in this function as the second argument to `Sprintf("%v/%v")`. So leading slash is redundant, and a trailing slash is not required, although not visible from just the function. Even the default value is "/api/v1", note the leading slash. So Trim() is used in exactly with that intention.

Reply:
Why do we need to take care of faulty configurations like `https://my.api.com///`? I'm fine with using `trimSuffix` to ease concat operations. But if we introduce such failure correction logic, it should be in place for other URLs as well.

Reply:
When I worked on this step I had two options:
* Refactor considerably to be able to give meaningful error messages in the log about parameters handed through from the step to pkg/fortify.
* Try to handle gracefully as much problems as possible.

In light of this, I don't see a point in handling one trailing slash, but not two or three. When the effective difference is using TrimRight() versus TrimSuffix(). A user configuring a URL with more than one trailing slash is probably a completely theoretical issue. And if I had a puzzling for-loop where I use TrimSuffix() until there are no more slashes, I would really see your point. I favor code which expresses the intend versus writing buggy code. The intend here is "the URL must not end in a slash, or we get a 503 later". If I write TrimSuffix(), the code does not achieve this.